src/app/dashboard/dashboard.component.html

```diff
@@ -19,6 +28,9 @@
         <div class="nav-panel-body">
           <a mat-list-item routerLink="/dashboard/homebrew/items">Items</a>
           <a mat-list-item routerLink="/dashboard/homebrew/spells">Spells</a>
+          <a mat-list-item href="https://critterdb.com/#/index" target="_blank">Creatures
```

Comment:
What is this?

Reply:
Any time you link to an external site with a target, it's good practice to also add `rel="noreferrer noopener"` to prevent the linked site from having JS access to the page that linked to it.

- `noopener` prevents linked pages from having access to the parent tab's window object via `window.opener`.
- `noreferrer` prevents the `Referrer` HTTP header from being sent and is a workaround for broader browser support (this also prevents the linked page from receiving referral info for analytics).

This site has a great explanation of the vulnerability as well as RTCs of the different ways of marking up links:
https://mathiasbynens.github.io/rel-noopener/

I use this ESLint rule for automatically validating this in react/jsx:
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md

Reply:
@zmckinnon: CritterDB is the external site you have to use to create homebrew monsters to import into Avrae - it was previously only linked to in `!help bestiary`